client/common/utils.tsx

```diff
@@ -964,10 +973,10 @@ export const statCalculators: { [key: string]: StatCalculator } = {
       ? getStatWithDefault(statsFromCustomSet, Stat.DAMAGE) +
         getStatWithDefault(statsFromCustomSet, Stat.AIR_DAMAGE)
       : 0,
-  [Stat.PODS]: (statsFromCustomSet) =>
+  [Stat.PODS]: statsFromCustomSet =>

```

Comment:
We should probably have a .prettierrc so formatting changes like this don't happen.. I'll look into it.